examples/multi-step-search-book-one-way-trip.py

```diff
@@ -0,0 +1,105 @@
+from datetime import date, timedelta
+from decimal import Decimal
+
+from duffel_api import Duffel
+
+
+if __name__ == "__main__":
+    print("Duffel Flights API - search, book and cancel example")
```

Comment:
**suggestion**: Change this to be more specific to this test scenario (that is, it's an MSS scenario).